### PR TITLE
Add ability to change background of text elements

### LINKED ISSUE #55: Add Text background color option
This issue has been migrated from mozilla/webmaker-android#1674.

It was originally written by [xmatthewx](https://api.github.com/users/xmatthewx) on Wed May 06 2015 and had the following description:

> Text on photos is tricky. I think we should add an option for background color on text. It will greatly increase what you can make with this app while we don't have shapes and stickers. 
> 
> Requirements:
> - ~~toggle on/off~~ palette with none option
> - standard tinker
> 
> Thoughts?
> 
> cc **@****thisandagain** **@****flukeout** **@****vazquez** 

On Wed May 06 2015, [flukeout](https://api.github.com/users/flukeout) commented:

> +1 on this idea, and the color bar that **@****vazquez** spec'd already has it:
> image

On Wed May 06 2015, [thisandagain](https://api.github.com/users/thisandagain) commented:

> :+1: Sounds like a good idea but we need to remove features from the critical path to v1 right now. Moving to backlog.

### PR DESCRIPTION
**Pull request by alanmoo**
And adjust i18n and markup to be consistent for link/text editor components

Closes #55

### CODE REVIEW

**Comment by alanmoo** [comment]
Also addresses #611

**Comment by gvn** [comment]
Looks like @vazquez has some updated colors for you in #611.

Also, I think it might be good to make a color module with named colors (similar to our colors.less, but a JS object). Something like:

``` javascript
module.exports = {
  blue: '#669eff',
  shadowBlue: '#5793f2',
  ...
};
```

This way we can re-use the colors throughout our UI code.

If we do this we'll need some new color names for the new picker palette. We should also add the new colors to the style guide.

**Comment by alanmoo** [comment]
I've updated this to address the simplified button colors in #720, also still closes #55.

File: src/pages/element/link-editor.jsx [hunk 1]
```diff
@@ -7,6 +7,9 @@ var api = require('../../lib/api');
 var platform = require('../../lib/platform');
 var types = require('../../components/basic-element/basic-element.jsx').types;
 
+// var backgroundColors = ['#9FD0E0', '#99CA47', '#EFC246', '#E06A2C', '#69A0FC', '#8173E4'];
```
Comment (gvn) [inline src/pages/element/link-editor.jsx:10]: Nuke this line?

**Comment by gvn** [comment]
One possible tweak...but R+!

File: src/pages/element/link-editor.jsx (hunk 1 above)
Comment (alanmoo) [inline src/pages/element/link-editor.jsx:10]: Wasn't sure if it was going to come back in once #719 was discussed further. I can remove it, but I was leaving it for some future reference.

Comment (gvn) [inline src/pages/element/link-editor.jsx:10]: Ok, it's fine to just leave it in that case...

**Pull request merged**